resetDevelopmentStack.sh

```diff
@@ -10,14 +10,19 @@ fi
 
 echo "Clearing Docker..."
 docker compose down --remove-orphans
-docker volume prune -f
+docker volume prune --filter "label!=keep" --force
 docker image prune -a -f
 
 echo "Clearing frontend clients..."
 ./gradlew clean
 ./gradlew assemble
 
 rm ./*github_env.log || true
+./build-utils/base_rebuild_gradle_dockerfile.sh
+set -o allexport
+source ./*github_env.log
+set +o allexportdocke
+
 find ./build-utils/ -name "rebuild*.sh" -exec bash -c 'eval "$1"' shell {} \;
```

Comment:
Was haltet ihr davon, beim lokalen start/resetDevStack die Production-Images nicht automatisch mit zu bauen. Das bauen der FrontendProd, BackendProd, E2ETest images nimmt einiges an Zeit in Anspruch und die werden ja eigentlich loakl nicht gebraucht oder?

Reply:
Fair. Finde ich gut